### PR TITLE
restrict gas price lower limit to 1 Gwei

### PR DESCRIPTION
**Pull request by SteveMieskoski**
### Bug

- [x] Updated CHANGELOG.md
- [x] Is this a user submitted bug?
  - [x] Link to issue: [twitter](https://twitter.com/taarushv/status/1164384221332303873)
- [x] Add PR label

### CODE REVIEW

File: src/components/SettingsModal/SettingsModal.vue [hunk 1]
```diff
@@ -269,12 +270,16 @@ export default {
...
-        ).toFixed();
+        if (new BigNumber(newVal).gte(1)) {
+          const toGwei = new BigNumber(
+            utils.toWei(`${newVal}`, 'gwei')
+          ).toFixed();
+          this.customGasEth = new BigNumber(
+            `${utils.fromWei(toGwei, 'ether')}`
+          ).toFixed();
+        } else {
+          this.customGas = 0;
```
Comment (kvhnuke) [inline src/components/SettingsModal/SettingsModal.vue]: set this to `this.customGas = 1;`

File: src/components/SettingsModal/SettingsModal.vue [hunk 2]
```diff
@@ -353,6 +358,7 @@ export default {
           ).toNumber()
         );
       } else {
+        if (!new BigNumber(this.customGas).gte(1)) return;
```
Comment (kvhnuke) [inline src/components/SettingsModal/SettingsModal.vue]: this shouldn't be necessary after the above change

Comment (SteveMieskoski) [reply src/components/SettingsModal/SettingsModal.vue]: made the change

File: src/components/SettingsModal/SettingsModal.vue (hunk 1 above)
Comment (SteveMieskoski) [reply src/components/SettingsModal/SettingsModal.vue]: made the change

**Pull request merged**